securedrop_export/export.py

```diff
@@ -359,19 +361,26 @@ def get_printer_uri(self):
         return printer_uri
 
     def install_printer_ppd(self, uri):
+        if "Brother" in uri:
+            self.printer_driver = BRLASER_DRIVER
+            self.printer_ppd = BRLASER_PPD
+        elif "LaserJet" in uri:
+            self.printer_driver = LASERJET_DRIVER
+            self.printer_ppd = LASERJET_PPD
```

Comment:
with this change, I can confirm that printing using your branch in securedrop-client and this version of securedrop-export all works without issue